transportable-udfs-trino/src/main/java/com/linkedin/transport/trino/TrinoFactory.java

```diff
@@ -55,17 +59,27 @@ public class TrinoFactory implements StdFactory {
   final FunctionBinding functionBinding;
   final FunctionDependencies functionDependencies;
   final Metadata metadata;
+  final TypeManager typeManager;
+  final FunctionManager functionManager;
+
+  final Session session;
 
   public TrinoFactory(FunctionBinding functionBinding, FunctionDependencies functionDependencies) {
```

Comment:
is it reasonable to assume that this exists for backward compatibility? 
should we mark this deprecated?